techlore/Plexus-app
# Update "Ma Banque (Crédit  Agricole)

### Description



Updates "Me Banque"
Needs locked bootloader
Play Service required to enable SécuriPass
Not tested with microG, estimated rating

### Testing



You need to be a Crédit Agricole Client

1. Open the App
2. Login to your account
3. SécuriPass will ask you to be enabled, process won't work without Play Services, the app will repeat a few times before working without SécuriPass
5. The rest works as expected


### Checklist
